typescript/cli/examples/warp-route-deployment-zksync.yaml

```diff
@@ -0,0 +1,21 @@
+# A config for a Warp Route deployment
+# Typically used with the 'hyperlane deploy warp' command
+#
+# Token Types:
+#   native
+#   collateral
+#   synthetic
+#
+# see comprehensive [list](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/sdk/src/token/config.ts#L8)
+---
+zksync1:
```

Comment:
This doesn't look like a valid deployment config. You would need at least 2 chains. Regardless, if you're not doing anything unique in this example, there's no need to add a new one